capability-providers/keyvalue/rust/src/generated.rs

```diff
@@ -0,0 +1,162 @@
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct PushArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+    #[serde(rename = "value")]
+    pub value: String,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct ListItemDeleteArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+    #[serde(rename = "value")]
+    pub value: String,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetAddArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+    #[serde(rename = "value")]
+    pub value: String,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetRemoveArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+    #[serde(rename = "value")]
+    pub value: String,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetUnionArgs {
+    #[serde(rename = "keys")]
+    pub keys: Vec<String>,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetIntersectionArgs {
+    #[serde(rename = "keys")]
+    pub keys: Vec<String>,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetQueryArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+}
+
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct KeyExistsArgs {
+    #[serde(rename = "key")]
+    pub key: String,
+}
+
+/// Response type for Get operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct GetResponse {
+    #[serde(rename = "value")]
+    pub value: String,
+    #[serde(rename = "exists")]
+    pub exists: bool,
+}
+
+/// Response type for Add operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct AddResponse {
+    #[serde(rename = "value")]
+    pub value: i32,
+}
+
+/// Response type for Delete operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct DelResponse {
+    #[serde(rename = "key")]
+    pub key: String,
+}
+
+/// Response type for list range operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct ListRangeResponse {
+    #[serde(rename = "values")]
+    pub values: Vec<String>,
+}
+
+/// Response type for list push operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct ListResponse {
+    #[serde(rename = "newCount")]
+    pub new_count: i32,
+}
+
+/// Response type for the Set operation, not to be confused with the set data
+/// structure
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetResponse {
+    #[serde(rename = "value")]
+    pub value: String,
+}
+
+/// Response type for set add operations
+#[derive(Debug, PartialEq, Deserialize, Serialize, Default, Clone)]
+pub struct SetOperationResponse {
+    #[serde(rename = "new_count")]
+    pub new_count: i32,
+}
```

Comment:
This may have been an oversight in the previous version, but we should pick a variable casing guideline and stick to it. I know in the httpserver example we already use camelCase for `queryString`, and personally I'm more of a fan of camelCase, but I'm not against snake.

Reply:
Definitely could've been an oversight. These things were generated using older tools. This likely wouldn't/won't show up until an AssemblyScript actor goes to make a set operation. Once we're generating, things like this should be more consistent.

Reply:
I'm for keeping with parity while we can with existing features 👍🏻